src/fetching/views/patchView.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultFetch } from '..';
+import { ZetkinView } from 'types/zetkin';
+
+export default function patchView(orgId: string | number, viewId: string | number, fetch = defaultFetch) {
+    return async (view: Record<string, unknown>):Promise<ZetkinView> => {
+        const url = `/orgs/${orgId}/people/views/${viewId}`;
+        const res = await fetch(url, {
+            body: JSON.stringify(view),
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            method: 'PATCH',
+        });
+        if (!res.ok) {
+            throw new Error(`Error making PATCH request to ${url}`);
```

Comment:
We have an `APIError` for this.

Reply:
I'll take a look! I just copied and pasted this code block from patchCampaign for my sins....